notebooks/quantum-machine-learning/encoding.ipynb

```diff
@@ -272,7 +272,7 @@
     "\n",
     "$$\\cssId{_}{|x\\rangle} = \\class{_big-o-times-n}{\\bigotimes^N_{i=1}} \\cos(x_i)|0\\rangle + \\sin(x_i)|1\\rangle$$\n",
     "\n",
-    "This is different from the previous two encoding methods, as it only encodes one data point at a time, rather than a whole dataset. It does however, only use $N$ qubits and a constant depth quantum circuit, making it amenable to current quantum hardware. \n",
+    "This is different from the previous two encoding methods, as it only encodes one data point at a time, rather than a whole dataset. It does however, only require N qubits or less and a constant depth quantum circuit, making it amenable to current quantum hardware. \n",
```

Comment:
```suggestion
    "This is different from the previous two encoding methods, as it only encodes one data point at a time, rather than a whole dataset. It does, however, only require N qubits or less and a constant depth quantum circuit, making it amenable to current quantum hardware. \n",
```